src/App.js

```diff
@@ -1,25 +1,81 @@
-import logo from './logo.svg';
 import './App.css';
+import foods from './foods.json';
+import { useState } from 'react';
+import FoodBox from './component/FoodBox';
+import AddFoodForm from './component/AddFoodForm';
+import SearchBar from './component/SearchBar';
+
+
+  function App() {
+    const [itens, setItens] = useState(foods);
+    const [showFood, setShowFood] = useState(foods);
+
+    const createFood = (newFood) => {
+      const food = [newFood, ...itens]
+      setItens(food)
+      setShowFood(food)
+    }
+
+    const filterFood = (searchQuery) => {
+    
+      let filteredFood = itens.filter((food) =>
+        food.name.toLowerCase().includes(searchQuery.toLowerCase())
+      );
+  
+      setShowFood(filteredFood);
+    }; 
+
+
+    const deleteFood = (foodName) => {
+    
+      let deletedFood = foods.filter((food) => food.name !== foodName)
```

Comment:
Here, you need to filter `itens`, just like you did before.

`let deletedFood = itens.filter((food) => food.name !== foodName)` instead of `foods.filter((food)`